DPG/Tasks/qaEfficiency.cxx

```diff
@@ -646,6 +693,7 @@ struct QaEfficiencyData {
   Configurable<int> phiBins{"phi-bins", 500, "Number of phi bins"};
   // Task configuration
   Configurable<bool> makeEff{"make-eff", false, "Flag to produce the efficiency with TEfficiency"};
+  Configurable<int> applyEvSel{"applyEvSel", 0, "Flag to apply rapidity cut: 0 -> no event selection, 1 -> Run 2 event selection, 2 -> Run 3 event selection"};
```

Comment:
I think here there is a copy-and-paste error in the info text

Reply:
@mario-krueger thanks! Good catch!